src/Forms/Shared/Samples/Map/ChangeBasemap/ChangeBasemap.xaml

```diff
@@ -1,15 +1,20 @@
 ﻿<?xml version="1.0" encoding="utf-8" ?>
-<ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
-             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"            
+<ContentPage x:Class="ArcGISRuntime.Samples.ChangeBasemap.ChangeBasemap"
+             xmlns="http://xamarin.com/schemas/2014/forms"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
              xmlns:esriUI="clr-namespace:Esri.ArcGISRuntime.Xamarin.Forms;assembly=Esri.ArcGISRuntime.Xamarin.Forms"
              xmlns:mapping="clr-namespace:Esri.ArcGISRuntime.Mapping;assembly=Esri.ArcGISRuntime"
-             x:Class="ArcGISRuntime.Samples.ChangeBasemap.ChangeBasemap">
-  <Grid>
-    <Grid.RowDefinitions>
-      <RowDefinition Height="*" />
-      <RowDefinition Height="auto" />
-    </Grid.RowDefinitions>
-    <esriUI:MapView x:Name="MyMapView" />
-    <Button x:Name="changeBasemapButton" Text="Change Basemap" Clicked="OnChangeBasemapButtonClicked" Grid.Row="1"/>
-  </Grid>
+             xmlns:toolkit="clr-namespace:Esri.ArcGISRuntime.Toolkit.Xamarin.Forms;assembly=Esri.ArcGISRuntime.Toolkit.Xamarin.Forms">
+    <Grid>
+        <Grid.RowDefinitions>
+            <RowDefinition Height="*" />
+            <RowDefinition Height="auto" />
+        </Grid.RowDefinitions>
+        <esriUI:MapView x:Name="MyMapView" />
+        <toolkit:BasemapGallery x:Name="MyBasemapGallery" BasemapSelected="BasemapSelected" />
```

Comment:
The transparent background can make the text hard to read. I would add an opaque background here.